playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -31,7 +31,7 @@
 
   - name: Drain Node for Kubelet upgrade
     command: >
-      {{ openshift.common.client_binary }} adm manage-node {{ openshift.common.hostname | lower }} {{ openshift.common.evacuate_or_drain }} --force
+      {{ openshift.common.admin_binary }} drain {{ openshift.common.hostname }} --force --delete-local-data --ignore-daemonsets
```

Comment:
@detiber `admin_binary` is my fault.  IIRC, I was having issues with with `oc adm drain` performing correctly at the time.  But since they are just linked to the `openshift` binary it was not making sense.